### DISCUSSION TITLE
How to set speech language?/如何设置语音？

### DISCUSSION THREAD

**Discussion by seanyth** [lencx/ChatGPT#567] [Q&A] [answered]
You can see the icons, but can't select anything. Is this like Microsoft Voice where you can choose between different voices or just one voice?
可以看到图标，但是无法选择。请问这个是像Microsoft语音那样可以选择不同语音还是只有一种声音？
![Screenshot 2023-03-06 083600](https://user-images.githubusercontent.com/95159727/222995844-2f68e26f-2c92-4ddb-8121-5490c5a7ac21.jpg)
Tampermonkey has a script to select different voices, so I have this confusion.
油猴上面有脚本可以选取不同语音，所以我有此困惑：https://greasyfork.org/scripts/459890-talkgpt/code/TalkGPT.user.js

**Comment by lencx** [comment, marked as answer]
它使用系统内置浏览器，可能是系统 webview 的问题？你可以尝试下载最新的 webview2 看是否能解决 https://developer.microsoft.com/en-us/microsoft-edge/webview2

**Reply by seanyth** [reply]
> 它使用系统内置浏览器，可能是系统 webview 的问题？你可以尝试下载最新的 webview2 看是否能解决 https://developer.microsoft.com/en-us/microsoft-edge/webview2

感谢，但是似乎并不是这个问题。我试图安装了Evergreen Bootstrapper，其提示安装失败，因为已经有安装2

**Reply by lencx** [reply]
下个版本修复它，有人贡献了代码 https://github.com/lencx/ChatGPT/pull/572
